gamedata.py

```diff
@@ -697,7 +715,7 @@ def keys(self) -> Generator[tuple[str, str | int], None, None]:
             if "blue_key_relic_skipped_log" in self._data:
                 yield ("Sapphire Key", self._data["blue_key_relic_skipped_log"]["floor"])
 
-    def _get_cards(self) -> Generator[tuple[str, dict[str, str]], None, None]:
+    def _get_cards(self) -> Iterable[tuple[str, dict[str, str]]]:
```

Comment:
You got it the wrong way around -- this function's return function was correct as a Generator. The argument on line 762 is the one that needs to be Iterable (ie the one that tells what argument we accept, which can be anything).